pkg/storage/azurecdn/storage.go

```diff
@@ -2,13 +2,19 @@ package azurecdn
 
 import (
 	"bytes"
-	"context"
 	"fmt"
 	"net/url"
 
+	"github.com/gomods/athens/pkg/storage"
+
 	"github.com/Azure/azure-storage-blob-go/2017-07-29/azblob"
+	"github.com/bketelsen/buffet"
+	"github.com/gobuffalo/buffalo"
 )
 
+// asserts that Storage implements storage.Saver
+var _ storage.Saver = &Storage{}
```

Comment:
no need for this, because the compiler will check for interface adherence where it's used

Reply:
Fixed locally